core/controllers/feedback.py

```diff
@@ -64,6 +64,18 @@ def post(self, exploration_id):
         self.render_json(self.values)
 
 
+class ThreadListHandlerForTopics(base.BaseHandler):
```

Comment:
Is this functionality live? If not should it be behind a flag?

Reply:
@nithusha21 ping?

Reply:
Done. It is hidden behind both ENABLE_GENERALIZED_FEEDBACK_THREADS and ENABLE_NEW_STRUCTURES